app/components/c-prices.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <div :class="b()">
+    <dl :class="b('price-definition')">
+      <dt :class="b('price-label')">{{ $t('c-prices.grossLabel') }}</dt>
+      <dd :class="b('price-value')">{{ displayPriceGross }}</dd>
+    </dl>
+    <dl :class="b('price-definition')">
```

Comment:
- [ ] I would recommend to do each price-definition dl with a v-if condition. In case of only a gros or net price will be delivered.

Reply:
Didn't know that could be the case. Added the v-if conditions.